Content.Server/Animals/Systems/UdderSystem.cs

```diff
@@ -134,4 +139,20 @@ private void AddMilkVerb(Entity<UdderComponent> entity, ref GetVerbsEvent<Altern
         };
         args.Verbs.Add(verb);
     }
+
+    private void OnExamine(Entity<UdderComponent> entity, ref ExaminedEvent args)
+    {
+        if (EntityManager.TryGetComponent(entity, out HungerComponent? hunger))
+        {
+            if (_hunger.GetHungerThreshold(hunger) == HungerThreshold.Overfed)
+                args.PushMarkup(Loc.GetString("udder-system-examine-overfed"));
+
+            if (_hunger.GetHungerThreshold(hunger) <= HungerThreshold.Peckish)
+                args.PushMarkup(Loc.GetString("udder-system-examine-hungry"));
+        }
+        else
+        {
+            args.PushMarkup(Loc.GetString("udder-system-examine-none"));
```

Comment:
This might be more elegant as a [relational switch expression](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#relational-patterns), but it's fine like this too. Just a comment.